src/test/java/core/basesyntax/TestAllMethods.java

```diff
@@ -0,0 +1,102 @@
+package core.basesyntax;
+
+import core.basesyntax.service.impl.CombineOutputImpl;
+import core.basesyntax.service.impl.CreateStrategyImpl;
+import core.basesyntax.service.impl.DataValidatorImpl;
+import core.basesyntax.service.impl.DoOperationImpl;
+import core.basesyntax.service.impl.ReadFromFileImpl;
+import core.basesyntax.service.impl.WriteToFileImpl;
+import core.basesyntax.service.interfaces.CombineOutput;
+import core.basesyntax.service.interfaces.CreateStrategy;
+import core.basesyntax.service.interfaces.DataValidator;
+import core.basesyntax.service.interfaces.DoOperation;
+import core.basesyntax.service.interfaces.ReadFromFile;
+import core.basesyntax.service.interfaces.WriteToFile;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestAllMethods {
```

Comment:
let's test all services separately and independently of each other

create a separate test class for testing each service